quickwit/quickwit-cli/src/split.rs

```diff
@@ -418,42 +405,44 @@ async fn describe_split_cli(args: DescribeSplitArgs) -> anyhow::Result<()> {
             )
         })?;
 
-    println!("{}", make_split_table(&[split_metadata], "Split"));
-
-    let split_file = PathBuf::from(format!("{}.split", args.split_id));
-    let (split_footer, _) = read_split_footer(index_storage, &split_file).await?;
-    let stats = BundleDirectory::get_stats_split(split_footer.clone())?;
-    let hotcache_bytes = get_hotcache_from_split(split_footer)?;
-
-    let mut file_rows = Vec::new();
-
-    for (path, size) in stats {
-        file_rows.push(FileRow {
-            file_name: path.to_str().unwrap().to_string(),
-            size: format_size(size, DECIMAL),
-        });
-    }
-    println!(
-        "{}",
-        make_table("Files in Split", file_rows.into_iter(), false)
-    );
-    if args.verbose {
-        let mut hotcache_files = Vec::new();
-        let hotcache_stats = HotDirectory::get_stats_per_file(hotcache_bytes)?;
-        for (path, size) in hotcache_stats {
-            hotcache_files.push(FileRow {
-                file_name: path.to_str().unwrap().to_string(),
-                size: format_size(size, DECIMAL),
-            });
-        }
-        let hotcache_table = make_table("Files in Hotcache", hotcache_files.into_iter(), false);
-        println!("{hotcache_table}");
-    }
+    println!("{}", make_split_table(&[split], "Split"));
+
+    // TODO: if we have access to the storage, we could fetch that.
```

Comment:
Maybe a sign that we need to persist the size of the split in the metastore when registering the split.

Reply:
We have it already in the metadata; I'm going to open an issue to track this.